### ISSUE TITLE
multiple plotting types for different data in same file

### ISSUE THREAD

**Issue by AndroidSim** [BioSystemsGroup/scripts#3]
For "cmp-plot.r" and similar plotting scripts: An issue with necrosis trigger and necrotic data is that cumulative measures should not be plotted with a moving average; however, "number of cells" and "mean distance" should. The difficulty is that these measures are in the same file. The plotting script should be modified to take this measurement format into account when plotting the data in these files.

**Comment by gepr** [comment]
Yes. Good point. But the script can't know which one should or shouldn't be done that way. I suspect what's needed is a version that compares (for `cmp-plot.r`) or plots (for `plot-cols.r`) a particular column as opposed to all columns. That way the type of plot (`raw|data|nodata`) combined with the particular column to plot would allow what we need. So, I suppose that calls for yet another plot script, maybe 2: `cmp-col.r` and `plot-col.r`.

**Comment by AndroidSim** [comment]
Ah, good point. Also, I guess we could check for a key in the column name, like "cumu" or something. The additional scripts passing the column name is a good solution.
